### PR TITLE
Update exception list summary api doc

### PR DESCRIPTION
**Pull request by ashokaditya**
Updates documentation for summary exception container. 
Related PR elastic/kibana/pull/123476

Preview [here](https://security-docs_1439.docs-preview.app.elstc.co/guide/en/security/master/exceptions-api-summary-exception-container.html).

### CODE REVIEW

**Review by benironside — APPROVED** [review]
LGTM

File: docs/detections/api/exceptions/api-summary-exception-container.asciidoc [hunk 1]
```diff
@@ -62,3 +64,17 @@ A summary of the exception container:
     total: 6,
 }
 --------------------------------------------------
+
+Note: For a host isolation exception the values for `windows`, `linux`, `macos` and `total` are all the same, as each host isolation exception entry applies to all the OSs.
```
Comment (jmikell821) [inline docs/detections/api/exceptions/api-summary-exception-container.asciidoc:68]:
```suggestion
NOTE: For a host isolation exception, the values for `windows`, `linux`, `macos`, and `total` are all the same, as each host isolation exception entry applies to all operating systems. 
```

**Comment by jmikell821** [comment]
Hey @ashokaditya! Just wanted to verify this is applicable to `v8.1.0`only and not `v8.0.0`?

**Review by joepeeples — APPROVED** [review]
LGTM, thanks!

**Comment by ashokaditya** [comment]
> Just wanted to verify this is applicable to v8.1.0only and not v8.0.0?

Yes. This is only for `8.1.0`.

**Pull request merged**